manifest.md

```diff
@@ -144,28 +138,33 @@ Unlike the [image index](image-index.md), which contains information about a set
 }
 ```
 
-## Example of a SCRATCH config or layer descriptor
+## Guidance for a filler descriptor
+
+*Implementers note*: The following is considered GUIDANCE for portability.
 
-Notice that the `mediaType` is subject to the usage or context, while the digest is specifically defined as `ScratchDigestSHA256`.
-When the `ScratchDigestSHA256` is used, the media type SHOULD be set to `application/vnd.oci.scratch.v1+json` to differentiate the descriptor from one pointing to content.
+Parts of the spec necessitate including a descriptor to a blob where some implementations of artifacts do not have associated content.
+While an empty blob (`size` of 0) may be preferable, practice has shown that not to be ubiquitously supported.
+The media type `application/vnd.oci.filler.v1+json` (`MediaTypeFiller`) has been specified for a descriptor that has no content for the implementation.
+The blob payload is the most minimal content that is still valid JSON object: `{}` (`size` of 2).
```

Comment:
```suggestion
The blob payload is the most minimal content that is still a valid JSON object: `{}` (`size` of 2, [base64](descriptor.md#embedded-content) of `e30=`).
```

Something like this? :eyes: (ref https://github.com/opencontainers/image-spec/pull/1068/files#r1207421094)

Reply:
I added a line indicating it was optional to minimize the risk someone will get confused.